audit/projects/k8s-infra-public-pii/services/enabled.txt

```diff
@@ -4,5 +4,7 @@ bigquerydatatransfer.googleapis.com  BigQuery Data Transfer API
 bigqueryreservation.googleapis.com   BigQuery Reservation API
 bigquerystorage.googleapis.com       BigQuery Storage API
 cloudresourcemanager.googleapis.com  Cloud Resource Manager API
+compute.googleapis.com               Compute Engine API
```

Comment:
What enabled compute?

Reply:
This is the consequence of a successful BigQuery Data Transfer job. Somehow the job will launch BigQuery jobs for the transfer. Like I explained in https://github.com/kubernetes/k8s.io/pull/2564, I accidentally recreated the job.